api/src/opentrons/drivers/smoothie_drivers/driver_3_0.py

```
@@ -913,17 +726,18 @@ def disengage_axis(self, axes: str):
             String containing the axes to be disengaged
             (e.g.: 'XY' or 'ZA' or 'XYZABC')
         """
-        axes = ''.join(set(axes.upper()) & set(AXES))
+        available_axes = set(AXES)
+        axes = ''.join(a for a in axes.upper() if a in available_axes)
```

Should we throw an exception here if we get an axis that we don't expect?